examples/default.yaml

```diff
@@ -0,0 +1,147 @@
+      network:
+      paging:
+      processes:
+      process:
+
+  ## Configuration for Filelog Receiver
+  ## Tails and parses logs from files
+  ## This instance is configured to handle syslog logs
+  ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver
+  filelog/syslog:
+    start_at: beginning
+    include_file_path_resolved: true
+    include:
+      - /var/log/auth.log
+      - /var/log/syslog
+      - /var/log/daemon.log
+      - /var/log/kern.log
+
+  ## Configuration for Filelog Receiver
+  ## Tails and parses logs from files
+  ## This instance is configured to handle dpkg logs
+  ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver
+  filelog/dpkg:
+    start_at: beginning
+    include_file_path_resolved: true
+    include:
+      - /var/log/dpkg.log
+
+exporters:
+  ## Configuration for Sumo Logic Exporter
+  ## This exporter supports sending logs, metrics and traces data to Sumo Logic.
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/exporter/sumologicexporter
+  sumologic:
+
+processors:
+  ## Configuration for Sumo Logic Schema Processor
+  ## The Sumo Logic Schema processor modifies the metadata on logs, metrics and traces sent to Sumo Logic
+  ## so that the Sumo Logic apps can make full use of the ingested data.
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/processor/sumologicschemaprocessor
+  sumologic_schema:
+
+  ## Configuration Sumo Logic Syslog Processor
+  ## The Sumo Logic Syslog processor can be used to create attribute with facility name based on facility code.
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/processor/sumologicsyslogprocessor
+  sumologic_syslog:
+
+  ## Configuration for Group by Attributes Processor
+  ## It groups records by attributes and move record attributes to resource attributes
+  ## ref: https://github.com/sumologic/opentelemetry-collector-contrib/tree/main/processor/groupbyattrsprocessor
+  groupbyattrs:
+    keys:
+      - log.file.name
+      - facility
+      - log.file.path_resolved
+
+  ## Configuration for Memory Limiter Processor
+  ## The memory_limiter processor is used to prevent out of memory situations on the collector.
+  ## ref: https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor
+  memory_limiter:
+  ## check_interval is the time between measurements of memory usage for the
+  ## purposes of avoiding going over the limits. Defaults to zero, so no
+  ## checks will be performed. Values below 1 second are not recommended since
+  ## it can result in unnecessary CPU consumption.
+  check_interval: 5s
+  ## Maximum amount of memory, in %, targeted to be allocated by the process heap.
+  limit_percentage: 75
+  ## Spike limit (calculated from available memory). Must be less than limit_percentage.
+  spike_limit_percentage: 20
```

Comment:
```suggestion
    ## check_interval is the time between measurements of memory usage for the
    ## purposes of avoiding going over the limits. Defaults to zero, so no
    ## checks will be performed. Values below 1 second are not recommended since
    ## it can result in unnecessary CPU consumption.
    check_interval: 5s
    ## Maximum amount of memory, in %, targeted to be allocated by the process heap.
    limit_percentage: 75
    ## Spike limit (calculated from available memory). Must be less than limit_percentage.
    spike_limit_percentage: 20
```